homeassistant/components/nintendo_parental/entity.py

```diff
@@ -0,0 +1,40 @@
+"""Base entity definition for Nintendo Parental."""
+
+from __future__ import annotations
+
+from pynintendoparental import Device
+
+from homeassistant.helpers.device_registry import DeviceEntryType, DeviceInfo
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import DOMAIN
+from .coordinator import NintendoUpdateCoordinator
+
+
+class NintendoDevice(CoordinatorEntity):
+    """Represent a Nintendo Switch."""
+
+    def __init__(
+        self, coordinator: NintendoUpdateCoordinator, device: Device, entity_id: str
```

Comment:
Rather not use the variable name entity id as it already has multiple use cases

Reply:
Thats not actually needed now as can get that from the entity description anyway so will remove.